src/IssueViz.Security/Taint/TaintIssuesSynchronizer.cs

```diff
@@ -75,6 +75,11 @@ public async Task SynchronizeWithServer()
             }
             try
             {
+                if (!sonarQubeService.IsConnected)
+                {
+                    return;
```

Comment:
I`d log that it wasn`t possible to sync otherwise it will be hard to diagnose what`s happening. I think it would be ok to log it with non-debug verbosity.

Also, this check doesn`t need to be inside the `try` block.